Sorting/BubbleSort.py

```diff
@@ -0,0 +1,18 @@
+nums = []
+print("Enter the Size of List: ")
+tot = int(input())
+
+print("Enter " + str(tot) + " Numbers: ")
+for i in range(tot):
+    nums.insert(i, int(input()))
+
+for i in range(tot-1):
```

Comment:
Explain the algo.

How did you test this program?

What is the complexity of this algo?